src/System.Memory/ref/System.Memory.cs

```diff
@@ -128,6 +131,9 @@ public static class MemoryExtensions
         public static int IndexOfAny(this ReadOnlySpan<byte> span, byte value0, byte value1, byte value2) { throw null; }
         public static int IndexOfAny(this ReadOnlySpan<byte> span, ReadOnlySpan<byte> values) { throw null; }
 
+        public static int LastIndexOf<T>(this ReadOnlySpan<T> span, T value) where T : IEquatable<T> { throw null; }
```

Comment:
Should we also have LastIndexOfAny for parity?

Reply:
Yes. I will add those in a later PR.

Reply:
Could you file an issue? Thanks.